src/encoded/schemas/changelogs/software.md

```diff
@@ -1,4 +1,7 @@
 ## Changelog for software.json
+### Minor changes since chema version 10
+
+* Added utility, proteomics, statistics, visualization, machine learning, integrated analysis, genetics to software_type.  Deprecated but did not remove purpose.
```

Comment:
Enums and properties should be italicized

Reply:
Also should intregated analysis be integrative analysis? If so, need update here and in json file.